bin/add-extension-package.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script to add a package to `packages` or `extensions`
+ */
+'use strict';
+
+const build = require('../packages/build');
+const path = require('path');
+const cwd = process.cwd();
+
+let name = process.argv[2];
+if (name == null) {
+  console.error(
+    'Usage: %s <package-name>',
```

Comment:
Please describe how is the script going to determine where to put the new package (`packages`, `extensions`, etc.).

Personally, I would like the script to support the following usage:

```
$ node bin/create-package extensions/my-cool-pkg
```